tests/test_parted_disk.py

```diff
@@ -88,11 +90,20 @@ def runTest(self):
         # TODO
         self.fail("Unimplemented test case.")
 
-@unittest.skip("Unimplemented test case.")
-class DiskAddPartitionTestCase(unittest.TestCase):
+class DiskAddPartitionTestCase(RequiresDisk):
+    """addPartition should return True if partition is added successfully(even
+    without committing)
+    """
     def runTest(self):
-        # TODO
-        self.fail("Unimplemented test case.")
+        self.disk.setFlag(parted.DISK_CYLINDER_ALIGNMENT)
+        flag = self.disk.getFlag(parted.DISK_CYLINDER_ALIGNMENT)
+        self.disk.setFlag(flag)
```

Comment:
why do we have setFlag, getFlag and setFlag again ? I don't quite understand the top 3 lines but the last 1 (even 2) seem redundant.